ha/core/event_analyzer/event_analyzerd.py

```diff
@@ -23,35 +23,93 @@
  ****************************************************************************
 """
 
+import os
 import time
 import sys
+import traceback
 
 from cortx.utils.log import Log
+from cortx.utils.conf_store.conf_store import Conf
+
+from ha import const
 from ha.core.config.config_manager import ConfigManager
+from ha.core.system_health.system_health import SystemHealth
+from ha.core.event_analyzer.watcher.watcher import Watcher
+
+class EventAnalyserService:
```

Comment:
"Analyzer" and "Analyser": both are used. Lets stick to one, I think Analyzer.

Reply:
yes 
updated to analyzer and Analyzer